fars/booking/models.py

```diff
@@ -11,6 +11,7 @@ class Bookable(models.Model):
     name = models.CharField(max_length=64)
     description = models.CharField(max_length=256)
     icon = models.CharField(max_length=32, default='tf.svg')
+    public = models.BooleanField(default=False)
```

Comment:
I'd prefer this to be "visibility" as some enum, so you can have different degrees of visibility, e.g. public, internal, private etc